src/test/java/org/apache/sysds/test/functions/federated/FederatedConstructionTest.java

```diff
@@ -142,7 +142,7 @@ public void federatedConstruction(Types.ExecMode execMode, String testFile, Stri
 			DMLScript.USE_LOCAL_SPARK_CONFIG = true;
 		}
 		fullDMLScriptName = HOME + testFile + ".dml";
-		programArgs = new String[] {"-args", "\"localhost:" + port + "/" + input(inputIdentifier) + "\"",
+		programArgs = new String[] {"-args", TestUtils.federatedAddress("localhost", port, input(inputIdentifier)),
```

Comment:
I Like this TestUtils for the federated Address. Could you change this for more of the federated tests?

Reply:
sure, I guess I should note it down in Jira and do a seperate PR?

Reply:
You don't have to do the change if you just make a Jira task and mark it as an warmup.

Reply:
How does one mark something as warmup? Do you mean with a label, because I would need to create a new one I guess? I added it as an improvement with low priority.